_data/studies.tsv

```diff
@@ -75,8 +75,12 @@ idr0066-voigt-mesospim	experimentC	prod70	853	3	0	3		0	4	3739	0.03797700596	3797
 idr0066-voigt-mesospim	experimentD	prod70	854	1	0	1		0	2	2978	0.04118653215	41186532145	6	6864.422024	5025 x 6735 x 1489 x 1 x 1
 idr0067-king-yeastmeiosis	experimentA	prod72	904	60	0	60		0	419	136360	0.2854708875	285470887482	1594	179.0908955	1224 x 1224 x 8 x 2 x 36
 idr0069-caldera-perturbome	screenA	prod71	2251	124	47616	47616			190464	571392	1.225666503	1225666502656	571912	2.143103314	1360 x 1024 x 1 x 3 x 1
+idr0070-kerwin-hdbr	experimentA	prod84	1104	287	0	287		0	4554	13512	4.9	1.33536E+11	9219	14484878.44	8223 x 9270 x 1 x 3 x 1
```

Comment:
`Size (TB)` vs `Size` do not seem to match for most studies ?

Reply:
Well, the reason for this is:
```
$ /opt/omero/server/OMERO.server/bin/omero fs usage Project:1104
Total disk usage: 133537132989 bytes in 9220 files
```
But:
```
$ python idr-utils/scripts/stats.py
...
 idr0070-kerwin-hdbr/experimentA                  | 1104    | 287 of 2362    | 0       | 4554    | 13512    | 4.9 TB
...
```

Reply:
Ah thanks. This comes back to which metric we want to use. I think the difference is that:

- `omero fs usage` is reporting the size of files on disk 
- the `stats.py` is only querying the DB and computing the bytes size from the image dimensions as well as the pixel type

The major difference between the two of them probably reflects file compression esp. for large volumes.

If we collectively agree the first metric should be used, I'll propose to open a PR against idr-utils to remove `Bytes` to clear the ambiguity.